pkg/backend/display/testdata/not-truncated/stack-output-resource-diff.json

```diff
@@ -0,0 +1,10 @@
+{"sequence":0,"timestamp":1677074634,"preludeEvent":{"config":{}}}
```

Comment:
To get this file, I…

1. ran `PULUMI_ENABLE_STREAMING_JSON_PREVIEW=true pulumi preview -s dev --json` in my repro project
2. pasted the output into this file
3. manually deleted all the diagnostic events to avoid clutter